modules/nf-core/yahs/main.nf

```diff
@@ -0,0 +1,36 @@
+process YAHS {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::yahs=1.2a.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/yahs:1.2a.2--h7132678_0':
+        'quay.io/biocontainers/yahs:1.2a.2--h7132678_0' }"
+
+    input:
+    tuple val(meta), path(bed)
+    path fasta
+    path fai
+
+    output:
+    tuple val(meta), path("*scaffolds_final.fa") , emit: scaffolds_fasta
+    tuple val(meta), path("*scaffolds_final.agp"), emit: scaffolds_agp
+    tuple val(meta), path("*bin")                , emit: binary
+    path "versions.yml"                                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
```

Comment:
@nvnieuwk meant that you needed to pass task.ext.prefix like [this](https://github.com/nf-core/modules/blob/ee80d14721e76e2e079103b8dcd5d57129e584ba/modules/nf-core/gsea/gsea/main.nf#L45) and [this](https://github.com/nf-core/modules/blob/ee80d14721e76e2e079103b8dcd5d57129e584ba/tests/modules/nf-core/gsea/gsea/nextflow.config#L15).

Reply:
Yes, my bad for not being clear enough about it :D

Reply:
@ksenia-krasheninnikova that might mean passing the prefix to the tool to determine output file names, or else renaming the files afterwards (as I had to do in the GSEA case).

Reply:
Thank you both! I hope the last update does it.